open-metadata-resources/open-metadata-archives/open-connector-archives/pom.xml

```diff
@@ -63,10 +63,10 @@
             <groupId>org.odpi.egeria</groupId>
             <artifactId>open-connector-framework</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.odpi.egeria</groupId>
-            <artifactId>cassandra-data-store-connector</artifactId>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
Can we delete these completely?

Reply:
Yes, just my bad habbit to keep history in comments :)  I will remove it.